crates/runtime/benches/bench_search/setup.rs

```diff
@@ -59,10 +69,10 @@ pub(crate) async fn setup_benchmark(
     Ok((rt, benchmark_result))
 }
 
-pub(crate) async fn load_search_queries(rt: &Runtime) -> Result<Vec<String>, String> {
+pub(crate) async fn load_search_queries(rt: &Runtime) -> Result<Vec<Query>, String> {
     let test_queries = rt
         .datafusion()
-        .query_builder("SELECT q._id, q.text, ARRAY_AGG(t.\"corpus-id\") AS corpus_ids FROM tests t JOIN test_query q ON t.\"query-id\" = q._id GROUP BY q._id, q.text ORDER BY q._id DESC")
```

Comment:
Why do we no longer need `corpus_ids`?

Reply:
The corpus_id retrieval has been moved to separate query

```
pub(crate) async fn load_query_relevance_data(rt: &Runtime) -> Result<QueryRelevance, String> {
    let test_queries = rt
        .datafusion()
        .query_builder(r#"SELECT "query-id", "corpus-id", score FROM test_score"#)
        .build()
        .run()
        .await
        .map_err(|e| format!("Failed to retrieve test scores: {e}"))?;

    let records = test_queries
        .data
        .try_collect::<Vec<RecordBatch>>()
        .await
        .map_err(|e| format!("Failed to retrieve test scores: {e}"))?;

    let qrels = extract_query_relevance_from_batches(&records)?;

    tracing::info!(
        "Loaded benchmark query relevance data for {num_rows} queries",
        num_rows = qrels.len()
    );

    Ok(qrels)
}

```